modules/demo/spatial/data.js

```diff
@@ -83,6 +83,9 @@ function loadFile(i) {
 
       let body = new Uint8Array(res.headers['content-length'] | 0);
 
+      console.log('content-encoding:', encoding);
+      console.log('content-length:', body.byteLength);
```

Comment:
I'm assuming we are keeping this logging in on purpose?

Reply:
oh woops, nope we don't need that